Detectors/FIT/FT0/workflow/src/FT0DataReaderDPLSpec.cxx

```diff
@@ -0,0 +1,80 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   FT0DataReaderDPLSpec.cxx
+
+#include "FT0Workflow/FT0DataReaderDPLSpec.h"
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace ft0
+{
+using namespace std;
+template <bool IsExtendedMode>
+void FT0DataReaderDPLSpec<IsExtendedMode>::init(InitContext& ic)
+
+{
+}
+template <bool IsExtendedMode>
+void FT0DataReaderDPLSpec<IsExtendedMode>::run(ProcessingContext& pc)
+
+{
+  DPLRawParser parser(pc.inputs());
+  mVecDigits.clear();
+  mVecChannelData.clear();
+  LOG(INFO) << "FT0DataReaderDPLSpec";
+  uint64_t count = 0;
+  for (auto it = parser.begin(), end = parser.end(); it != end; ++it) {
+    //Proccessing each page
+    count++;
+    //auto rdhPtr = it.get_if<o2::header::RAWDataHeader>(); //PROBLEM!
+    //auto rdhPtr = it.get_if<o2::header::RAWDataHeaderV4>();
+    auto rdhPtr = it.get_if<o2::header::RAWDataHeaderV6>();
```

Comment:
what is the problem? The default ``RAWDataHeader`` is aliased to RAWDataHeaderV6, and it is better to use default than hardcoded V6.

Reply:
Now it's okay, I will change it to default RAWDataHeader.